Controller/Cards/ListAction.php

```diff
@@ -0,0 +1,46 @@
+<?php
+namespace Omise\Payment\Controller\Cards;
+
+use Magento\Customer\Model\Session;
+use Magento\Framework\App\Action\Context;
+use Magento\Framework\App\RequestInterface;
+use Magento\Framework\App\ResponseInterface;
+use Magento\Framework\Exception\NotFoundException;
+
+class ListAction extends \Magento\Framework\App\Action\Action
+{
+    /**
+     * @var \Magento\Customer\Model\Session
+     */
+    protected $customerSession;
+
+    /**
+     * @var \Magento\Framework\View\Result\PageFactory
+     */
+    private $pageFactory;
+
+    /**
+     * @param Context $context
+     * @param Session $customerSession
+     */
+    public function __construct(
+        Context $context,
+        Session $customerSession,
+        \Magento\Framework\View\Result\PageFactory $pageFactory
+    ) {
+        parent::__construct($context);
+        $this->customerSession = $customerSession;
+        $this->pageFactory = $pageFactory;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function execute()
+    {
+        $resultPage = $this->pageFactory->create();
+        $resultPage->getConfig()->getTitle()->set(__('Stored Credit/Debit Cards'));
+
+        return $resultPage;
+    }
+}
```

Comment:
@jacstn I see now you completely removed the dispatch method, so we don't need to check if `!$this->customerSession->authenticate()` anymore or it has been done by Magento as default already?